source/Background/Controller.js

```diff
@@ -154,15 +159,19 @@ backgroundController.exposeController('isConnected', async (opts, url) => secure
     }
   });
 }));
+*/
 
+/* DEPRECATED
```

Comment:
should we just delete this?

Reply:
yep, I was just leaving it for reference but we should delete this for sure